config/initializers/foreman.rb

```diff
@@ -28,10 +28,12 @@
 Dashboard::Loader.load
 
 # clear our users topbar cache
-begin
-  User.unscoped.pluck(:id).each do |id|
-    Rails.cache.delete("views/tabs_and_title_records-#{id}")
+if defined?(User)
```

Comment:
Yeah, I'm fine with leaving it with a `defined?` call only, but I'd suggest putting a comment next to it instead to make the intention clear.